pkg/api/unversioned/deep_copy_generated.go

```diff
@@ -21,127 +21,15 @@ limitations under the License.
 package unversioned
 
 import (
-	"time"
-
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	time "time"
 )
 
-func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {
```

Comment:
@wojtek-t if you don't have time tomorrow morning I'll take a look at this.

On May 23, 2016, at 8:35 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/api/unversioned/deep_copy_generated.go
<https://github.com/kubernetes/kubernetes/pull/26051#discussion_r64312478>:

>  )
>
> -func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c *conversion.Cloner) error {

I will try to look at his PR tonight.
… <#>
On Mon, May 23, 2016 at 11:52 AM, Wojciech Tyczynski < ***@***.***> wrote:
In pkg/api/unversioned/deep_copy_generated.go <#26051 (comment)
<https://github.com/kubernetes/kubernetes/pull/26051#discussion_r64265280>>
: > ) > > -func DeepCopy_unversioned_APIGroup(in APIGroup, out *APIGroup, c
*conversion.Cloner) error { I talked with @lavalamp
<https://github.com/lavalamp> <https://github.com/lavalamp> and he said
that he has a PR in-flight (that needs a rebase) that will separate
registration from the generation. Maybe we can wait for that one? — You are
receiving this because you were mentioned. Reply to this email directly or
view it on GitHub <
https://github.com/kubernetes/kubernetes/pull/26051/files/ee463c073a7f6853fc2a6019de435aa5d4e6c2c4#r64265280>


—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/26051/files/ee463c073a7f6853fc2a6019de435aa5d4e6c2c4#r64312478>